### PR TITLE
Added doc content for workflow

### PR DESCRIPTION
**Pull request by pjacobsam**

### CODE REVIEW

File: components/common/query/realm.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+The request must include the following query parameters:
+
+| Field          | Type   | Description                        |
+| -------------- | ------ | ---------------------------------- |
+| Realm          | String | Defines the specific context or domain within which the Clearing-House request is being made. This header determines the environment or operational scope for the request. Allowed values include `ABS` (Automated Banking Systems), `CHM` (Clearing House Management), and `BCH` (Bank Clearing House). Each value represents a different area of functionality or service. |
```
Comment (blaisefonseca) [inline components/common/query/realm.md]:
```suggestion
| Realm          | String | Defines the specific context or domain within which the Clearing-House request is being made. This header determines the environment or operational scope for the request. Allowed values include `ABS` (Access and Benefit-sharing Clearing House), `CHM` (Clearing House Mechanism), and `BCH` (Biosafety Clearing House). Each value represents a different area of functionality or service. |
```

Comment (blaisefonseca) [reply components/common/query/realm.md:5]: `suffix the realms with `-dev` to use the development and. `-trg` for training `

File: components/common/query/realm.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+
+**Realm Values for Development:**
```
Comment (blaisefonseca) [inline components/common/query/realm.md:8]: make sure to highlight that the api url will be api.cbddev.xyz

File: components/ircc/workflow/request-body.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+The request body should be a JSON object containing the following fields:
+
+| Field           | Type          | Description   | Required |
+| --------------- | ------------- | ------------- | -------- |
+| `document`      | Object        | The schema object that represents the document being submitted. For example, the [IRCC schema](/ircc/schema). | Yes      |
```
Comment (blaisefonseca) [inline components/ircc/workflow/request-body.md]:
```suggestion
| `document`      | Object        | The schema object that represents the body of the record type. For example, the [IRCC schema](/ircc/schema). | Yes      |
```

File: components/ircc/workflow/request-body.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+| `additionalInfo`| String        | Additional information related to the document. This field can hold a maximum of 1000 characters.    | No       |
```
Comment (blaisefonseca) [inline components/ircc/workflow/request-body.md]: this field is available only when a use with NAU role is submitting a request to the PA to publish a record.

File: components/ircc/workflow/request-error.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+| Error Type             | Description                           |
+| ---------------------- | ------------------------------------- |
+| Error.MissingRealmParam     | `realm` param is missing, please provide valid realm in querystring      |
```
Comment (blaisefonseca) [inline components/ircc/workflow/request-error.md]: or httpHeaders

File: components/ircc/workflow/request-error.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+| Error.RealmNotConfigured      | Realm ${realm} is not configured, please provide valid realm.          |
+| Error.SchemaNotConfigured     | Schema not configured for the given realm ${realm}.                    |
+| Error.AuthorizationRequired   | 403: Authorization required, user is anonymous.                        |
```
Comment (blaisefonseca) [inline components/ircc/workflow/request-error.md]:
```suggestion
| Error.AuthorizationRequired   | 401: Authorization required, user is anonymous.                        |
```

Comment (pjacobsam) [reply components/ircc/workflow/request-error.md]: image

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**